src/bot/controllers/SearchController.js

```diff
@@ -7,17 +8,40 @@ class SearchController extends BaseController {
             this.i18n('search.whatAreYouLooking', [$.sender.username]),
             [
                 this.base.reply.postback(this.i18n('search.categories.breakfast'), 'SEARCH_LOOKING_FOR_BREAKFAST'),
-                this.base.reply.postback(this.i18n('search.categories.lunch'), 'SEARCH_LOOKING_FOR_LUNCH'),
-                this.base.reply.postback(this.i18n('search.categories.dinner'), 'SEARCH_LOOKING_FOR_DINNER'),
-                this.base.reply.postback(this.i18n('search.categories.coffeeAndTee'), 'SEARCH_LOOKING_FOR_COFFEE_AND_TEA'),
-                this.base.reply.postback(this.i18n('search.categories.nightlife'), 'SEARCH_LOOKING_FOR_NIGHTLIFE'),
-                this.base.reply.postback(this.i18n('search.categories.thingsToDo'), 'SEARCH_LOOKING_FOR_THINGS_TO_DO'),
+                this.base.reply.postback(this.i18n('search.categories.diner'), 'SEARCH_LOOKING_FOR_DINER'),
+                this.base.reply.postback(this.i18n('search.categories.coffee'), 'SEARCH_LOOKING_FOR_COFFEE'),
+                this.base.reply.postback(this.i18n('search.categories.fastfood'), 'SEARCH_LOOKING_FOR_FASTFOOD'),
+                this.base.reply.postback(this.i18n('search.categories.restaurant'), 'SEARCH_LOOKING_FOR_RESTAURANT'),
+                this.base.reply.postback(this.i18n('search.categories.bar'), 'SEARCH_LOOKING_FOR_BAR'),
             ]
         );
     }
 
-    renderPlaces($) {
-        const category = $.quickReplyPayload.replace('SEARCH_LOOKING_FOR_', '');
+    async renderPlaces($) {
+        let category = $.quickReplyPayload.replace('SEARCH_LOOKING_FOR_', '');
+        switch (category) {
```

Comment:
Let's move with one to the object an retrieve needed category by key
For example
```
const categoryObject = {
  BREAKFAST: 'breakfast',
  COFFEE: 'coffee show',
  FAST_FOOD: 'fast food',
  ...
}
```

and then retrieve like `categoryObject[category]`